Source/BoundaryConditions/FieldBoundaries.H

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2025 Luca Fedeli
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "Utils/WarpXAlgorithmSelection.H"
+
+#include <AMReX_Array.H>
+#include <AMReX_Config.H>
+
+#include <utility>
+
+namespace warpx::boundary_conditions
+{
+    /**
+     * \brief Reads the field boundary conditions from
+     * the inputfile
```

Comment:
This can probably fit on one line?
```suggestion
     * \brief Reads the field boundary conditions from the input file
```